### ISSUE TITLE
urine osteopontin terms

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3269]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal urine cytokine level
2. abnormal urine osteopontin level
3. decreased urine osteopontin level
4. increased urine osteopontin level

>2. Synonyms (e.g., absent spleen)

2. abnormal urine Spp1 level; abnormal urine OSP level; abnormal urine OPN level
3. reduced urine osteopontin level; decreased urine Spp1 level, decreased urine OSP level, decreased urine OPN level
4. increased urine Spp1 level, increased urine OSP level, increased urine OPN level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. deviation from the normal urinary levels of any of the non-antibody proteins made by inflammatory leukocytes and some non-leukocytic cells that affect the behavior of other cells
2. deviation from the normal urinary levels of osteopontin, a cytokine that upregulates expression of interferon-gamma and interleukin-12
3. reduction in the amount in the urine of osteopontin, a cytokine that upregulates expression of interferon-gamma and interleukin-12
4. increase in the amount in the urine of osteopontin, a cytokine that upregulates expression of interferon-gamma and interleukin-12

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:29437896

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal urine protein level (MP:0006315)
2. abnormal urine cytokine level (new)
3. abnormal urine osteopontin level (new)
4. abnormal urine osteopontin level (new)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0002-9553-7227
--

**Comment by sbello** [comment]
ref that osteopontin is a cytokine - https://pubmed.ncbi.nlm.nih.gov/8262332/ PMID: 8262332

**Comment by sbello** [comment]
MP:0020993 - cytokine
MP:0020994 - abnormal osteopontin
MP:0020995 - decreased
MP:0020996 - increased
